### PR TITLE
Remove "*:port" & improve SSH_BIND docs

### PR DESCRIPTION
**Pull request by foosel**
As promised in #27, my suggestion on how to fix the README regarding the non working "*:port" example and `SSH_BIND_IP`.

### CODE REVIEW

**Review by jnovack — CHANGES_REQUESTED** [review]
So far so good!  Just a few minor corrections.

File: README.md [hunk 1]
```diff
@@ -211,17 +209,26 @@ Defines how the tunnel will be set up:
 
 #### SSH_BIND_IP
 
+You can define which IP address the tunnel will use to bind on *remote* 
+(SSH_MODE of `-R`) or *local* (SSH_MODE of `-L`). The default
+is `127.0.0.1` only.
+
+##### SSH_MODE of `-R` (default)
```
[the remark this answers is not in the document]
Comment (foosel) [reply README.md:216]: I'm not sure I understand you correctly. 

The thing is that - as far as I understood the documentation available before and the source - the implications of `SSH_BIND_IP` are dependent on the setting of `SSH_MODE`, which is documented a couple lines up (line 203 and following). 

My goal here was to make that clear by separating the documentation of **`SSH_BIND_IP`** into two sub sections relating to the active **`SSH_MODE`** setting. In the case of the default mode of `-R` the big warning applies. In the case of `-L` you actually might want to bind the port to everything and hence that's explained.

Should I maybe clarify this duality further?

> #### SSH_BIND_IP
> You can define which IP address the tunnel will bind `SSH_TUNNEL_PORT` on. The default is `127.0.0.1`. The `SSH_BIND_IP` setting has different implications depending on whether you have an `SSH_MODE` value of `-R` (default) or `-L` specified, read on for those.
> 
> [...]

[the remark this answers is not in the document]
Comment (jnovack) [reply README.md:216]: If the function of `SSH_BIND_IP` changes based on `SSH_MODE`, then the "parent" should be `SSH_MODE`, and the "child" should be `SSH_BIND_IP`.

[the remark this answers is not in the document]
Comment (jnovack) [reply README.md:216]: Wait, sorry, I didn't read it in context.  Was attempting to occupy a toddler and didn't view the file entirely.

**Pull request merged**